src/TestLibrary/TestClass.cs

```diff
@@ -17,5 +17,13 @@ public void TestMethod()
             }
             2.ToString();
         }
+
+        public void Testmethod2()
+        {
+            for (int i = 5 - 1; i >= 0; i--)
+            {
+            }
```

Comment:
SA1513 - Statements or elements wrapped in curly brackets must be followed by a blank line.